app/views/files/index.slim

```diff
@@ -19,6 +19,8 @@ ul
 					input type="hidden" name="_method" value="DELETE"
 					button type="submit" Delete
 
+				a href="#{file.route}?download=true" Download
+
```

Comment:
Should we maybe change the edit to /edit or ?edit and make the direct link download?